src/Misc/layoutbin/en-US/strings.json

```diff
@@ -562,5 +562,6 @@
   "WindowsLogonAccountNameDescription": "User account to use for the service",
   "WindowsLogonPasswordDescription": "Password for the account {0}",
   "WorkFolderDescription": "work folder",
-  "Y": "Y"
+  "Y": "Y",
+  "ArtifactNotFromLinkedDefintion": "The build {0} does not belong to the linked build definition {1}"
```

Comment:
you need to sort the string ids in alphabetical order. one of the l0 tests will catch this. pl. make sure you run l0 tests and they pass with this fix.